app/controllers/v0/evss_claims_controller.rb

```diff
@@ -24,5 +24,11 @@ def request_decision
       claim.update_attributes(requested_decision: true)
       render_job_id(jid)
     end
+
+    private
+
+    def skip_sentry_exception_types
+      EXCEPTIONS_WITH_RECORD_NOT_FOUND
```

Comment:
isn't this just inherited? why list it if it is inherited from application controller?

Reply:
the application controller method has a different array of exceptions

Reply:
I see, but wouldn't it be better to subtract from that array or add to that array in this method, just to make it more readable? In other words:

super + [some_new_error_to_add]

or

super - [some_new_error_to_remove]

Reply:
NITPICK